src/Framework/Bootstrap/GacelaConfig.php

```diff
@@ -228,13 +231,21 @@ public function disableEventListeners(): self
         return $this;
     }
 
+    /**
+     * Register a generic listener when any event happens.
```

Comment:
Maybe a better description would be helpful (take a look at the review comment), I guess more people will have the same questions as I do 😃

Reply:
I added some more doc lines below.